nemo_skills/pipeline/train.py

```diff
@@ -219,8 +219,9 @@ def train(
     )
 
     with run.Experiment(expname) as exp:
+        prev_job_id = None
```

Comment:
It's not really id, but a task object, so better to rename, but not too important

Reply:
Ah ok will do

Reply:
Fixed